addons/figshare/README.md

```diff
@@ -33,7 +33,7 @@ has instructions for use with ngrok, but there are other methods.
 from addons.figshare.client import FigshareClient
 from website.oauth.models import ExternalAccount
 
-me = User.find_one(Q('username', 'eq', '<your username>'))
+me = OSFUser.load('<osf_guid>')
```

Comment:
OSFuser needs to be imported 'from osf.models.user import OSFUser', otherwise  works fine